src/Microsoft.DotNet.Build.Tasks/GenerateTestOutputDirList.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GenerateTestOutputDirList : Task
```

Comment:
Nothing seems to use this Task. is it necessary? Or should it not be part of this change